test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -273,4 +276,27 @@ public void clusterResizingTest2() throws Exception {
 
         shutdownCorfuServer(p0);
     }
+
+    @Test
+    public void forceRemoveTwoDeadNodes() throws Exception {
+        Harness harness = new Harness();
+
+        int numNodes = 3;
+        List<Node> nodeList = harness.deployCluster(numNodes);
+        Node n0 = nodeList.get(0);
+        Node n1 = nodeList.get(1);
+        Node n2 = nodeList.get(2);
+
+        CorfuRuntime rt = new CorfuRuntime(n2.getClusterAddress()).connect();
+
+        assertThat(rt.getLayoutView().getLayout().getAllServers().size()).isEqualTo(numNodes);
+
+        // Shutdown two nodes
+        run(n0.shutdown, n1.shutdown);
+
+        rt.getManagementView().forceRemoveNode(n0.getAddress(), workflowNumRetry, timeout, pollPeriod);
+        rt.getManagementView().forceRemoveNode(n1.getAddress(), workflowNumRetry, timeout, pollPeriod);
+
+        assertThat(rt.getLayoutView().getLayout().getAllServers().size()).isEqualTo(1);
```

Comment:
could you also make sure the correct expected layout is present.

Reply:
done.